Makefile.toml

```diff
@@ -64,6 +64,15 @@ REPO_METADATA_EXPIRING_WITHIN = "3 days"
 # You can use ALLOW_CLOBBER=true with the `ssm` task to make it overwrite existing values.
 # (This is not required with `promote-ssm` because the intent of promotion is overwriting.)
 
+# This can be overridden to provide a custom import spec for a VMware OVA.
+# Using configuration from Infra.toml, we substitute the correct value for
+# network, and whether or not to mark a VM as a template
+VMWARE_IMPORT_SPEC_PATH = "${BUILDSYS_ROOT_DIR}/tools/pubsys/support/vmware/import_spec.template"
+
+# You can set VMWARE_DATACENTERS to override the list of datacenters from
+# Infra.toml for VMware commands; it's a comma-separated list like
+# "datacenter1,datacenter2"
```

Comment:
Is this comment in the right place?

Reply:
Indeed - It closely mimics the `AWS_REGIONS` comment a few lines up.